docs/arc42/08_concepts.md

```diff
@@ -20,6 +20,9 @@ the respective material number of a partner leading to the following constellati
     - own material number = material number supplier
     - material number partner = material number customer
 
+Furthermore, as of ItemStockSAMM v2.0.0 the supplier CX ID is the sole identifier for a material of a respective partner.
+Thus, the material global asset id is the same as the partner material CX ID of the supplier.
```

Comment:
Refrase second: Within the shared asset approach, there are two scenarios:
- when acting as a customer, the supplier's global asset id (Catena-X ID) is leading and taken from the PartTypeInformation of the supplier's twin.
- when acting as a supplier, the initially generated global asset id (Catena-X ID) is handed over to the customers.

Within PURIS this results in the following steps:
- When creating a material (independent of the role) a global asset id (Catena-X ID) is defined on the material, which is used for the supplier twin.
- When creating a mpr, the global asset id (Catena-X ID)
  - may be set externally (controller),
  - is taken over from the suppliers PartTypeInformation (acting as supplier) OR
  - is set to the material's catena-X id (acting as customer)